### PR TITLE
add header

### LINKED ISSUE #4: Create header
TBD:
1. Logo
2. In header name of the current place
3. Search button
4. Should be responsive
5. (new) Should be variant for details page
Image

### PR DESCRIPTION
**Pull request by Avareco**
image
image

### CODE REVIEW

File: src/App.svelte
Comment (oleksandrstarov) [inline src/App.svelte:8]: Maybe some styles to set up position for header is required too?

File: src/components/Header.svelte
Comment (oleksandrstarov) [inline src/components/Header.svelte]: Maybe 'isSearchContext'? Or I am missing something?

Comment (Avareco) [reply src/components/Header.svelte]: We will merge this PR after this one [https://github.com/oleksandrstarov/svelte/pull/7](url) so I remove this variable

File: src/App.svelte
Comment (Avareco) [reply src/App.svelte:8]: > Maybe some styles to set up position for header is required too?

Not sure that I get you. What kind of styles do you mean to I add?

**Comment by Avareco** [comment]
Noticed that I used wrong task number in the name of the branch, will rename before merge

File: src/App.svelte
Comment (Avareco) [reply src/App.svelte:8]: Done, sticky position added

**Review by oleksandrstarov — APPROVED** [review]
Good job

**Review by Aleshchenk0 — APPROVED** [review]

**Pull request merged**
